results/Snowflake__snowflake-arctic-embed-l-v2.0/edc2df7b6c25794b340229ca082e7c78782e6374/model_meta.json

```diff
@@ -0,0 +1 @@
+{"name": "Snowflake/snowflake-arctic-embed-l-v2.0", "revision": "edc2df7b6c25794b340229ca082e7c78782e6374", "release_date": "2024-12-04", "languages": ["afr_Latn", "ara_Arab", "aze_Latn", "bel_Cyrl", "bul_Cyrl", "ben_Beng", "cat_Latn", "ceb_Latn", "ces_Latn", "cym_Latn", "dan_Latn", "deu_Latn", "ell_Grek", "eng_Latn", "spa_Latn", "est_Latn", "eus_Latn", "fas_Arab", "fin_Latn", "fra_Latn", "glg_Latn", "guj_Gujr", "heb_Hebr", "hin_Deva", "hrv_Latn", "hat_Latn", "hun_Latn", "hye_Armn", "ind_Latn", "isl_Latn", "ita_Latn", "jpn_Jpan", "jav_Latn", "kat_Geor", "kaz_Cyrl", "khm_Khmr", "kan_Knda", "kor_Hang", "kir_Cyrl", "lao_Laoo", "lit_Latn", "lav_Latn", "mkd_Cyrl", "mal_Mlym", "mon_Cyrl", "mar_Deva", "msa_Latn", "mya_Mymr", "nep_Deva", "nld_Latn", "pan_Guru", "pol_Latn", "por_Latn", "que_Latn", "ron_Latn", "rus_Cyrl", "sin_Sinh", "slk_Latn", "slv_Latn", "som_Latn", "sqi_Latn", "srp_Cyrl", "swe_Latn", "swa_Latn", "tam_Taml", "tel_Telu", "tha_Thai", "tgl_Latn", "tur_Latn", "ukr_Cyrl", "urd_Arab", "vie_Latn", "yor_Latn", "zho_Hans"], "n_parameters": 568000000, "memory_usage": null, "max_tokens": 8192.0, "embed_dim": 1024, "license": "apache-2.0", "open_weights": true, "public_training_data": null, "public_training_code": null, "framework": ["Sentence Transformers", "PyTorch"], "reference": "https://huggingface.co/Snowflake/snowflake-arctic-embed-l-v2.0", "similarity_fn_name": "cosine", "use_instructions": false, "training_datasets": null, "adapted_from": "BAAI/bge-m3-retromae", "superseded_by": null, "loader": "sentence_transformers_loader"}
```

Comment:
```suggestion
{"name": "Snowflake/snowflake-arctic-embed-l-v2.0", "revision": "edc2df7b6c25794b340229ca082e7c78782e6374", "release_date": "2024-12-04", "languages": ["afr_Latn", "ara_Arab", "aze_Latn", "bel_Cyrl", "bul_Cyrl", "ben_Beng", "cat_Latn", "ceb_Latn", "ces_Latn", "cym_Latn", "dan_Latn", "deu_Latn", "ell_Grek", "eng_Latn", "spa_Latn", "est_Latn", "eus_Latn", "fas_Arab", "fin_Latn", "fra_Latn", "glg_Latn", "guj_Gujr", "heb_Hebr", "hin_Deva", "hrv_Latn", "hat_Latn", "hun_Latn", "hye_Armn", "ind_Latn", "isl_Latn", "ita_Latn", "jpn_Jpan", "jav_Latn", "kat_Geor", "kaz_Cyrl", "khm_Khmr", "kan_Knda", "kor_Hang", "kir_Cyrl", "lao_Laoo", "lit_Latn", "lav_Latn", "mkd_Cyrl", "mal_Mlym", "mon_Cyrl", "mar_Deva", "msa_Latn", "mya_Mymr", "nep_Deva", "nld_Latn", "pan_Guru", "pol_Latn", "por_Latn", "que_Latn", "ron_Latn", "rus_Cyrl", "sin_Sinh", "slk_Latn", "slv_Latn", "som_Latn", "sqi_Latn", "srp_Cyrl", "swe_Latn", "swa_Latn", "tam_Taml", "tel_Telu", "tha_Thai", "tgl_Latn", "tur_Latn", "ukr_Cyrl", "urd_Arab", "vie_Latn", "yor_Latn", "zho_Hans"], "n_parameters": 568000000, "memory_usage": null, "max_tokens": 8192.0, "embed_dim": 1024, "license": "apache-2.0", "open_weights": true, "public_training_data": false, "public_training_code": false, "framework": ["Sentence Transformers", "PyTorch"], "reference": "https://huggingface.co/Snowflake/snowflake-arctic-embed-l-v2.0", "similarity_fn_name": "cosine", "use_instructions": true, "training_datasets": null, "adapted_from": "BAAI/bge-m3-retromae", "superseded_by": null, "loader": "sentence_transformers_loader"}
```
Seems like the model does use instruction (though very simple instructions):

```py
# from hf page
# ...
query_embeddings = model.encode(queries, prompt_name="query") 
# ...
```

Reply:
Good point! If you want, I can also open a PR updating this value in `mteb/models` since the rest of arctic models, as well as other models like the `bge` family, also use basic instructions for encoding the query (the prompt used to be: _"Represent this sentence for searching relevant passages: "_)

Reply:
Would love that

Reply:
Perfect, I'll do it next week 👌